README.md

```diff
@@ -3,6 +3,85 @@ JupyterLab Extension Builder
 
 Tools for building JupyterLab extensions.
 
+An extension to JupyterLab is a module that provides one or more
+plugins to the JupyterLab application.  This library provides
```

Comment:
A JupyterLab extension provides additional, optional functionality to JupyterLab's built-in capabilities. The extension is a module that provides one or more plugins to the JupyterLab application. To streamline third-party development of extensions, this library...